project/CodeCoverageSettings.scala

```diff
@@ -29,7 +29,8 @@ object CodeCoverageSettings {
     ".*Routes.*",
     "config.*",
     "testOnly.*",
-    "testOnlyDoNotUseInAppConf.*"
+    "testOnlyDoNotUseInAppConf.*",
+    "shared.config.*"
```

Comment:
Locally I got over 95% for coverage for this branch, can check this again when fully migrated if you prefer